backend/pyproject.toml

```diff
@@ -50,6 +50,7 @@ dependencies = [
     "unstract-workflow-execution @ file:///${PROJECT_ROOT}/../unstract/workflow-execution",
     "azure-identity==1.16.0",
     "azure-mgmt-apimanagement==3.0.0",
+    "google-cloud-recaptcha-enterprise>=1.19.0",
```

Comment:
@tahierhussain why do we add this dependency and not use it anywhere in the OSS code?

Reply:
@chandrasekharan-zipstack As of now, I don't believe we support adding a dependency inside the cloud's backend. Let's discuss this with @hari-kuriakose during our next call and find a way to manage this situation.

Reply:
@tahierhussain Let's remove any dependencies which are not required in OSS.

Then, let's add a `requirements.txt` containing those Enterprise only dependencies, and try to load them via:
```
pdm lock --override requirements.txt
```
Please check https://pdm-project.org/latest/usage/dependency/#via-cli-option. Let's see if this suits us.

**NOTE**: This will need changes in Enterprise Dockerfiles and CI workflows.

Reply:
@tahierhussain  @hari-kuriakose I hope these dependencies reside along with the plugin that's written and not in this repo.

Reply:
@chandrasekharan-zipstack Done. The changes have been made in the cloud repository.